### PR TITLE
🏷️ fix organization types and dtos

### PR DESCRIPTION
**Pull request by AntoineKM**

### CODE REVIEW

File: packages/node/src/rest/types/organizations/index.ts [hunk 1]
```diff
@@ -45,14 +45,22 @@ export enum OrganizationSocialType {
 export type OrganizationMember = {
   user: User;
   role: OrganizationMemberRole;
+  status: OrganizationMemberStatus;
```
Comment (jerembdn) [inline packages/node/src/rest/types/organizations/index.ts:48]: Donc si un membre est pas accepté, il est quand même dans la liste mais avec le status "rejected" ?

Comment (AntoineKM) [reply packages/node/src/rest/types/organizations/index.ts:48]: So when we search our member list, we also see those who have rejected the invitation?

Comment (AntoineKM) [reply packages/node/src/rest/types/organizations/index.ts:48]: We could use this to suggest that Admins invite the person again.

Comment (jerembdn) [reply packages/node/src/rest/types/organizations/index.ts:48]: je changerai pas ton pdv

**Review by jerembdn — APPROVED** [review]
super cool

**Pull request merged**